### PR TITLE
readme and dir.props updates for 2.7.2 version and nuget.org package

### PR DESCRIPTION
**Pull request by tomjebo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,12 +35,15 @@ See License.txt in the project root for license information.
...
+The Release Package on Nuget.org
+================================
+The official release NuGet packages for Open XML SDK are available on Nuget.org at https://www.nuget.org/packages/DocumentFormat.OpenXml. 
 
 The Latest Builds
 =================
 
-## Where to get the NuGet package?
-The NuGet package for Open XML SDK is currently available as a custom feed on MyGet. You can trust this package source, since the custom feed is locked and only this project feeds into the source.
+## Where to get the latest build NuGet package?
```
Comment (twsouthwick) [inline README.md]: This reads a little weird - maybe "Where to get the NuGet packages of the latest builds"?

File: README.md [hunk 2]
```diff
@@ -53,7 +56,9 @@ There is a known issue in WindowsBase that causes crashes when handling large da
 | .NET Standard | NuGet                    | .NET Core 1.0 |
 
 ## How to install the NuGet package?
-The package you want to install is DocumentFormat.OpenXml. See https://dotnet.myget.org/gallery/open-xml-sdk 
+The package you want to install is DocumentFormat.OpenXml. 
+
+See https://www.nuget.org/packages/DocumentFormat.OpenXml for the release packages or https://dotnet.myget.org/gallery/open-xml-sdk for the latest build packages.
```
Comment (twsouthwick) [inline README.md]: What about:

NuGet packages are available for [release builds](https://www.nuget.org/packages/DocumentFormat.OpenXml) or [CI Builds](https://dotnet.myget.org/gallery/open-xml-sdk) and follow [semantic versioning](http://www.semver.org).

File: README.md [hunk 3]
```diff
@@ -89,7 +94,8 @@ Change Log
 ==========
 
 Version 2.7.2 : June 6, 2017
-- Fixed assembly versioning.
+- Fixed assembly versioning
```
Comment (twsouthwick) [inline README.md]: Maybe a bit more explicit: Fixed issue where assembly version wasn't set in assembly

File: README.md (hunk 2 above)
Comment (tomjebo) [inline README.md]: I do like that better.

File: README.md (hunk 3 above)
Comment (tomjebo) [inline README.md]: sure

**Comment by tomjebo** [comment]
@twsouthwick PR is updated.

**Comment by tomjebo** [comment]
Fixes #210

**Pull request merged**